packages/app-staking/src/Actions/Account/index.tsx

```diff
@@ -46,23 +47,25 @@ interface StakeState {
   isStashNominating: boolean;
   isStashValidating: boolean;
   nominees?: string[];
-  sessionIds: string[];
+  sessionKeys: string[];
   stakers?: Exposure;
   stakingLedger?: StakingLedger;
   validatorPrefs?: ValidatorPrefs;
 }
 
+interface DerivedStakingAccount extends DerivedStakingInfo, DerivedSessionKeyInfo {};
+
 function toIdString (id?: AccountId | null): string | null {
   return id
     ? id.toString()
     : null;
 }
 
-function getStakeState (allAccounts: string[], allStashes: string[] | undefined, { controllerId: _controllerId, nextSessionIds, nominators, rewardDestination, sessionIds, stakers, stakingLedger, validatorPrefs }: DerivedStakingAccount, stashId: string, validateInfo: ValidatorInfo): StakeState {
+function getStakeState (allAccounts: string[], allStashes: string[] | undefined, { controllerId: _controllerId, nextSessionKeys, nominators, rewardDestination, sessionKeys, stakers, stakingLedger, validatorPrefs }: DerivedStakingAccount, stashId: string, validateInfo: ValidatorInfo): StakeState {
   const isStashNominating = !!(nominators?.length);
-  const isStashValidating = !validateInfo[1].isEmpty || !!allStashes?.includes(stashId);
-  const nextConcat = u8aConcat(...nextSessionIds.map((id): Uint8Array => id.toU8a()));
-  const currConcat = u8aConcat(...sessionIds.map((id): Uint8Array => id.toU8a()));
+  const isStashValidating = !validateInfo.isEmpty || !!allStashes?.includes(stashId);
```

Comment:
Why was this change required: `!validateInfo[1].isEmpty` -> `!validateInfo.isEmpty` ? 💭

Reply:
api.query.staking.nominators for a stashId will always return me Option of Map and not Option of array.. Map is a structure where I can check if it is empty or not..